en/docs/setup/migrating-data-publishers.md

```diff
@@ -52,25 +52,37 @@ and migrating it to use the
 [AnalyticsSessionDataPublishHandler](https://github.com/wso2-extensions/identity-data-publisher-authentication/blob/master/components/org.wso2.carbon.identity.data.publisher.authentication.analytics.session/src/main/java/org/wso2/carbon/identity/data/publisher/authentication/analytics/session/AnalyticsSessionDataPublishHandler.java)
 instead.
 
-1.  Open the `           identity.xml          ` file found in the
-    `           <IS_HOME>/repository/conf/identity          ` folder and
-    remove the listener configuration corresponding to the data
-    publisher that you wish to migrate. For example, the following
-    listener configuration corresponding to the
-    [DASSessionDataPublisherImpl](https://github.com/wso2-support/identity-data-publisher-authentication/blob/support-5.1.7/components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/impl/DASSessionDataPublisherImpl.java)
-    has been removed in order to deprecate it.
+1.  Open the `identity.xml.j2` file found in the
```

Comment:
Is it correct to use this file?

Reply:
It's a j2 file and I think it's correct since we need to add a template to that file

Reply:
We are not recommending to change .j2 file, it a particular configuration is not templated, we need to provide a WUM and fix in the next version

Reply:
Fixed: 755e715a1bdd5ac9f01e6a73d74dc83761780cad